### PR TITLE
Twilio mail

### LINKED ISSUE #232: Reset employee password through email
- send email to employees with appropriate URL
- have page with one time access in backend
- implement button in emp login

### PR DESCRIPTION
**Pull request by amurph24**
email is integrated:
- need to install a few packages in the backend see readme under nodemailer for full list
- also need to update env file

Password reset for admins is also working now, close #232 :
- need to install 'promise' in backend

### CODE REVIEW

**Comment by amurph24** [comment]
The password reset structure in the frontend should probably be streamlined a little bit, but for now it is functional

File: MERN Scripts/backend/models/resetToken.js
Comment (camwlsn) [inline MERN Scripts/backend/models/resetToken.js:1]: Great use of token generation

**Review by camwlsn — APPROVED** [review]
Great work, this is an important feature to have

**Review by SpeiserK — APPROVED** [review]
Great updates!

**Review by zachpren0 — APPROVED** [review]
Excellent work Implementing the password reset feature

**Pull request merged**